packages/cli/src/commands/lint.js

```diff
@@ -21,8 +23,23 @@ export const builder = (yargs) => {
 }
 
 export const handler = ({ fix }) => {
-  execa('yarn eslint', [fix && '--fix', 'web/src', 'api/src'].filter(Boolean), {
-    cwd: getPaths().base,
+  const paths = getPaths()
+  const webExists = fs.existsSync(paths.web.src)
+  const apiExists = fs.existsSync(paths.api.src)
+  const options = []
+
+  if (fix) {
+    options.push('--fix')
+  }
+  if (webExists) {
+    options.push('web/src')
+  }
+  if (apiExists) {
+    options.push('api/src')
+  }
```

Comment:
Thanks for this:

another way of writing this would be:
```suggestion
  [
     fix && '--fix',
     webExists && 'web/src',  
     apiExists && 'api/src',
  ].filter(Boolean)
```

Reply:
Thanks for the feedback. I've made those changes.